src/query/service/src/interpreters/interpreter_select.rs

```diff
@@ -231,6 +233,10 @@ impl Interpreter for SelectInterpreter {
 
         // 0. Need to build physical plan first to get the partitions.
         let physical_plan = self.build_physical_plan().await?;
+        let query_plan = physical_plan
+            .format(self.metadata.clone(), SharedProcessorProfiles::default())?
+            .format_pretty()?;
+        warn!("Query plan: \n{}", query_plan);
```

Comment:
how about make it a info level ?

Reply:
also, we can consider attach a query id in this log, thus may make it easier for us to diagnose the related query plan for a failed query

Reply:
> how about make it a info level ?

That will cause a lot of logs in the unit test.

Reply:
`warn` is considered to be a higher level than `info`, wouldn't `warn` introduce logs into the unit test?

Reply:
> also, we can consider attach a query id in this log, thus may make it easier for us to diagnose the related query plan for a failed query

Done.

Reply:
> `warn` is considered to be a higher level than `info`, wouldn't `warn` introduce logs into the unit test?

You're right, let me fix it.